[USING] minor rework of ELI5 (#774)
zkSNACKs/WasabiDoc: docs/using-wasabi/ELI5.md

Choose a very secure password, as this encrypts the secrets, and should not be guessed by others.
You will always need this password in order to spend your bitcoins, so take great care to back it up properly.
In the next step, Wasabi will show your 12 recovery words.

> These must also be carefully backed up, as they are needed to access your bitcoins on another computer or in another wallet software.

Reviewer: ```suggestion
These must also be carefully backed up, as they are needed to recover your wallet (together with the password) on another computer or in another wallet software.
```

Make sure you store the backup of your recovery words in a different location than the backup of your password.

:::warning Backup both!
If you ever need to restore your wallet in Wasabi, or any other wallet, you will need both: your password AND your 12 recovery words.
:::

See [this chapter](/using-wasabi/WalletGeneration.md) for more information on how to securely generate your wallet.

## Receiving bitcoin

> In order to receive bitcoins, you can generate an address in the `Receive` tab.

Reviewer: ```suggestion
In order to receive bitcoins, you should generate an address in the `Receive` tab.
```
